restaurants/views.py

```diff
@@ -1,3 +1,132 @@
-from django.shortcuts import render
+import json, re, bcrypt, jwt
 
-# Create your views here.
+from django.views           import View
+from django.http            import JsonResponse
+from django.db.models       import Avg, Q
+from django.core.exceptions import FieldError
+
+from restaurants.models     import Category, Location, Menu, Restaurant
+from reviews.models         import Review, ReviewImage
+from users.models           import User, WishList
+from users.utils            import login_decorator
+class RestaurantDetailView(View):
+    def get(self, request, restaurant_id):
+        try:
+            request.user = None
```

Comment:
아래 코드가 의미하는게 무엇인지 설명해주실 수 있나요?
```suggestion
            request.user = None
```

Reply:
가고싶다가 True인지 False인지 나타내는 코드에서 유저정보를 가져와야되는데 이 페이지는 비회원도 볼 수 있게 되어있는 페이지라서 로그인되어있지않다면 False라고 표시하려고 넣었습니다!